spec/rails_helper.rb

```diff
@@ -8,6 +8,7 @@
 end
 
 require "rspec/rails"
+require "active_record"
```

Comment:
Why was this necessary?

Reply:
When switching from `attr_accessor` to `attribute` I'm getting errors from shoulda (originating [here](https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/rails_shim.rb#L51)) matchers unless ActiveRecord is present.

Reply:
I think you're maybe just missing an `include ActiveModel::Attributes` on your signup class?

Reply:
Ah fair enough, weird the step models work but this one doesn't 🤔 I can't see any differences between them though 🤷

Reply:
Nor me, I spent ages comparing them side by side 🤷🏽‍♂️